docs/source/operations/overview.rst

```diff
@@ -23,6 +23,16 @@ For each component section below, configuration options relates to the component
 
 .. _cert-manager: https://cert-manager.io
 
+.. note::
```

Comment:
Should be on the[ lib installation page](https://docs.substra.org/en/latest/substrafl_doc/substrafl_overview.html#installation) and not on the deployment page?